--- a/builds/unix/configure
+++ b/builds/unix/configure
@@ -20124,23 +20124,6 @@ Library configuration:
   pthread:       $have_pthread
 " >&6;}
 
-# Warn if docwriter is not installed
-
-if test $have_docwriter = no; then
-  { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING:
-  \`make refdoc' will fail since pip package \`docwriter' is not installed.
-  To install, run \`$PIP install docwriter', or to use a Python
-  virtual environment, run \`make refdoc-venv' (requires pip package
-  \`virtualenv').  These operations require Python >= $PYTHON_MIN_VERSION.
-  " >&5
-printf "%s\n" "$as_me: WARNING:
-  \`make refdoc' will fail since pip package \`docwriter' is not installed.
-  To install, run \`$PIP install docwriter', or to use a Python
-  virtual environment, run \`make refdoc-venv' (requires pip package
-  \`virtualenv').  These operations require Python >= $PYTHON_MIN_VERSION.
-  " >&2;}
-fi
-
 # Warn if pthread is not available
 
 if test $have_pthread = no; then
--- a/builds/unix/configure.ac
+++ b/builds/unix/configure.ac
@@ -1215,17 +1215,6 @@ Library configuration:
   pthread:       $have_pthread
 ])
 
-# Warn if docwriter is not installed
-
-if test $have_docwriter = no; then
-  AC_MSG_WARN([
-  `make refdoc' will fail since pip package `docwriter' is not installed.
-  To install, run `$PIP install docwriter', or to use a Python
-  virtual environment, run `make refdoc-venv' (requires pip package
-  `virtualenv').  These operations require Python >= $PYTHON_MIN_VERSION.
-  ])
-fi
-
 # Warn if pthread is not available
 
 if test $have_pthread = no; then
